pkg/distro/rhel8/partition_tables.go

```diff
@@ -423,3 +247,188 @@ var edgeBasePartitionTables = distro.BasePartitionTableMap{
 		},
 	},
 }
+
+func getEc2PartitionTables(osVersion string, isRHEL bool) distro.BasePartitionTableMap {
+	if osVersion == "8.9" {
+		// 8.9: has 500 MiB /boot
+		return distro.BasePartitionTableMap{
```

Comment:
(nitpick) I would probably extract this into a `getEc2PartitionTablesRH8_9_only()` helper to make this function a bit smaller. Something like:
```go
if osVersion == "8.9" {
		// 8.9: has 500 MiB /boot
		return getEc2PartitionTablesRH8_9_only()
}
```